### DISCUSSION TITLE
[Q] Adjust image position in cell

### DISCUSSION THREAD

**Discussion by tiagocostaHB** [exceljs/exceljs#2383] [Q&A]
Hi, 

I'm adding some images to my cells, and managed to adapt the cells to have the same width and height of the image, but now images are covering the border of the of the cell, so I want the images inside each cell to go one point to the the right and one point to the bottom, how can I do this?

This is how I'm adding images currently:
`    worksheet.addImage(imageId2, {
      tl: { col: (dataRow.length - 1), row: rowIndex + 2 },
      ext: { width: 93, height: 93 }
    });
`

Thanks.

**Comment by skypesky** [comment]
@tiagocostaHB Here is my solution, Maybe it works for you.
```typescript
const range = {
    tl: { col: 2 - 0.94, row: 10 - 0.75 },
    br: { col: 6, row: 12 }
}
worksheet.addImage(imageId, range);
```

**Comment by Ks89** [comment]
With the latest stable release of the library, I cannot add both `tl: { col: .., row: ... }` and `br: { col: .., row: ... }`, because typescript type Anchor requires also other fields like nativeCol, nativeColOff, nativeRow, nativeRowOff. This isn't documented in the official readme.
